tasks/upgrade.yml

```diff
@@ -77,16 +77,18 @@
           Please return to continue...
 
     # NOTE: Only one set of syspaths packages can be installed at the same
-    # time.
+    # time. 
+    # NOTE: In Ansible 2.11, support for squash_actions on a package module 
+    # will be removed. We repeat the prefix for each package to avoid a loop:
+    # https://docs.ansible.com/ansible/latest/porting_guides/porting_guide_2.7.html#using-a-loop-on-a-package-module-via-squash-actions
     - name: Remove the previous syspaths packages
       package:
-        name: "{{ postgresql_upgrade_old_scl_name }}-{{ item }}"
+        name:
```

Comment:
Is `with_items` deprecated? Because in this case the old version looks more readable.

Reply:
Squash actions is deprecated. With squash actions removed, with_items will be inefficient for tasks like package or yum. I agree that this way it is less readable, but alternatives are constructing dynamic list as a separate variable, which would further reduce readability.